src/core/Akka.Streams/Serialization/StreamRefSerializer.cs

```diff
@@ -37,52 +36,51 @@ public sealed class StreamRefSerializer : SerializerWithStringManifest
         public StreamRefSerializer(ExtendedActorSystem system) : base(system)
         {
             _system = system;
-            _serialization = system.Serialization;
```

Comment:
This is the actual fix. `system.Serialization` is null if config were added before the ActorSystem is started because all serializer plugins were instantiated _before_ `ActorSystem.Serialization` is assigned.

Reply:
This could make for a decent analyzer rule